src/components/LogRow/RowRenderer/index.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  CellMeasurer,
+  CellMeasurerCache,
+  ListRowProps,
+  ListRowRenderer,
+} from "react-virtualized";
+import { LogTypes } from "constants/enums";
+import { AnsiiRow } from "../AnsiiRow";
+import { ResmokeRow } from "../ResmokeRow";
+
+interface RowProps {
+  listRowProps: ListRowProps;
+  data: {
+    getLine: (index: number) => string | undefined;
+    wrap: boolean;
+    processedLines: (number | number[])[];
+  };
+}
```

Comment:
Kind of wondering if this belongs better in `BaseRow/index.tsx` or the `types.ts` file in this folder

Reply:
Good call done